test/utility/parse_opts.c

```diff
@@ -120,6 +217,18 @@ parse_tiered_opt(TEST_OPTS *opts)
     case 'T':
         opts->tiered_storage = true;
         break;
+    case 'd':
```

Comment:
Alphabetize these in the switch statement.

Reply:
Fixed in 0e6b43406c158379072274241f2c15de26c317b5